src/devtools/client/debugger/src/components/SecondaryPanes/Frames/NewFrames.tsx

```diff
@@ -102,14 +103,20 @@ function FramesRenderer({
         }
       })}
       <Suspense fallback={<div className="pane-info empty">Loading async frames…</div>}>
-        <FramesRenderer panel={panel} pauseId={pauseId} asyncIndex={asyncIndex + 1} />
+        <FramesRenderer
+          panel={panel}
+          pauseId={pauseId}
+          asyncIndex={asyncIndex + 1}
+          copyStackTrace={copyStackTrace}
+        />
       </Suspense>
     </>
   );
 }
 
 export default function Frames({ panel }: { panel: "debugger" | "console" | "webconsole" }) {
-  const sourcesLoading = useAppSelector(getSourcesLoading);
+  const state = useAppSelector(state => state);
```

Comment:
:stop_sign: **DON'T DO THIS!**  It will cause the component to re-render on _every single dispatched action!**

A better option would be to put that logic into a thunk, which has access to `getState`

Reply:
D'oh! And the `FramesRenderer` component made the same mistake...